### PR TITLE
Replace Blacklist & Whitelist with Blocklist & Safelist

### PR DESCRIPTION
**Pull request by mrunalk**

### CODE REVIEW

File: rollbar/__init__.py [hunk 1]
```diff
@@ -252,7 +252,7 @@ def _get_pylons_request():
         'safe_repr': True,
         'scrub_varargs': True,
         'sizes': DEFAULT_LOCALS_SIZES,
-        'whitelisted_types': []
+        'safelisted_types': []
```
Comment (waltjones) [inline rollbar/__init__.py:255]: This looks publicly accessible. => "configure by calling init() or overriding directly"

Any concern about breaking existing users?

Comment (brianr) [reply rollbar/__init__.py:255]: 👍 I'd expect this change (and the ones in other repos) to be done in a backwards-compatible way. We can deprecate the old param names and phase them out under our deprecation policy (which we can write now if we don't already have one).

**Comment by mmaday** [comment]
FYI, many places are making a similar change and the [terminology is scattered](https://english.stackexchange.com/questions/51088/alternative-term-to-blacklist-and-whitelist).  Twitter just announced they are going with the terms `Denylist` and `Allowlist`: https://mobile.twitter.com/TwitterEng/status/1278733303508418560.

**Comment by mrunalk** [comment]
Closing this as Letania is working on #354

**Pull request closed without merging**